elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemParser5_0.java

```diff
@@ -28,11 +37,42 @@
  */
 public class ElytronSubsystemParser5_0 extends ElytronSubsystemParser4_0 {
 
+    final PersistentResourceXMLDescription jaspiConfigurationParser = PersistentResourceXMLDescription.builder(PathElement.pathElement(JASPI_CONFIGURATION))
+            .setXmlWrapperElement(JASPI)
+            .addAttributes(JaspiDefinition.ATTRIBUTES)
```

Comment:
This is not holding the PR but we should discourage referencing attributes collections when defining a XML parser.

If another attribute is added to the `JaspiDefinition.ATTRIBUTES` in a later version, it would be picked up by the XML 5.0 when it must not.

Most other Elytron's `PersistentResourceXMLDescription` explicitly enumerates the attribute instead of relying on their `ATTRIBUTES` collection.

Reply:
I think what tends to happen is many start out referencing the Attributes array at the outset - then once we add more attributes we split both versions into new and old.